task2-3/src/types/Group.ts

```diff
@@ -0,0 +1,7 @@
+import { Permission } from './Permission';
+
+export type Group = {
+    id: string;
+    name: string;
+    permissions: Array<Permission>
```

Comment:
On all my past projects and on my current project we use this syntax: `Permission[]`

Reply:
Actually, it is a single place where I used exactly this form. It should be the same form everywhere in code) fixed.